emailer.py

```diff
@@ -2,15 +2,15 @@
 from email.header import Header
 from email.mime.multipart import MIMEMultipart
 from email.mime.text import MIMEText
-from email import Charset
+from email import charset
```

Comment:
email.charset is a 'legacy' API but I don't understand how the new API works so I'm leaving it alone for now.

Reply:
Is it being `legacy` related to the case change?

Reply:
Sort of -- in Python 2, `Charset` is a class, and in Python 3 it's been moved to a module called `charset`. But they've moved other classes to modules without also deprecating them.